RecCalorimeter/tests/options/runSiPMDualReadout.py

```diff
@@ -0,0 +1,115 @@
+from Gaudi.Configuration import *
+from Configurables import ApplicationMgr
+
+from Configurables import k4DataSvc
+dataservice = k4DataSvc("EventDataSvc", input="testIDEA_o1_v03.root")
+
+from Configurables import PodioInput
+podioinput = PodioInput("PodioInput",
+    collections = [
+        "DRcaloSiPMreadout_scint",
+        "DRcaloSiPMreadoutRawHit",
+        "DRcaloSiPMreadoutTimeStruct",
+        "DRcaloSiPMreadoutWaveLen"
+    ],
+    OutputLevel = DEBUG
+)
+
+from Configurables import SimulateSiPMwithEdep
+sipmEdep = SimulateSiPMwithEdep("SimulateSiPMwithEdep",
```

Comment:
Event though they are the default, can you explicitly set `inputHitCollection` here?